bucket/dngrep.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.9.482",
+    "description": "Powerful search for Windows",
+    "homepage": "http://dngrep.github.io/",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/dnGrep/dnGrep/releases/download/v2.9.482.0/dnGREP.2.9.482.x64.msi",
+            "hash": "59cadfa678eca183ebdd9840455da3b3ac9872eb69b39d8175bc6594ebe0532f"
+        },
+        "32bit": {
+            "url": "https://github.com/dnGrep/dnGrep/releases/download/v2.9.482.0/dnGREP.2.9.482.x86.msi",
+            "hash": "ffe7136771bd90fb884587b89341a64a2fd2b0cc4008e55222cd496e47a9d813"
+        }
+    },
+    "extract_dir": "dnGREP",
+    "bin": "dnGREP.exe",
```

Comment:
Any commandline arguments?

Reply:
the commandline arguments are simply settings to open dnGREP gui with.
What should I do with them?

Reply:
Can you show a link to them (any docs/screenshots etc.)

Reply:
https://github.com/dnGrep/dnGrep/wiki/Command-Line

Reply:
Okay, nothing to change here in that case